test_data/cri.log

```diff
@@ -0,0 +1,2 @@
+2016-10-06T00:17:09.669794202Z stdout P log content 1
```

Comment:
A couple of more example log lines that don't contain sensitive information would be beneficial here

Reply:
Done